src/store/flow/block/set-contact-property.ts

```diff
@@ -0,0 +1,76 @@
+import {IRootState} from '@/store'
+import {findBlockOnActiveFlowWith as findBlock, IBlock, IContext, SetContactProperty} from '@floip/flow-runner'
+import {ActionTree, MutationTree} from 'vuex'
+import {IFlowsState} from '../index'
+
+export interface ISetContactPropertyIndex {
+  blockId: IBlock['uuid'],
+  key: SetContactProperty['property_key'],
+}
+
+export interface ISetContactPropertyIndexWithValue extends ISetContactPropertyIndex {
+  value: SetContactProperty['property_value'],
+}
+
+export const mutations: MutationTree<IFlowsState> = {
+  block_addContactProperty(state, {blockId, key, value}: ISetContactPropertyIndexWithValue) {
+    const block = findBlock(blockId, state as unknown as IContext)
+
+    // Ensure the block has a set_contact_property[]
+    if (!block.config.set_contact_property) {
+      block.config.set_contact_property = []
+    }
+
+    // We should avoid duplicates
+    if (block.config.set_contact_property.find(({property_key}) => property_key === key)) {
+      throw new Error(`set_contact_property[${key}] already exists`)
```

Comment:
Would it be benefitial to add a one-line doc to `block_addContactProperty` about this?
So that wherever we consume it, we know that it might throw an error.